cloud_run_service_pubsub/main.tf

```diff
@@ -34,6 +34,7 @@ resource "google_cloud_run_service_iam_binding" "binding" {
 
 # [START cloudrun_service_pubsub_token_permissions]
 resource "google_project_iam_binding" "project" {
+  project = "PROJECT_ID"   # project - (Required) The project id of the target project. This is not inferred from the provider.
```

Comment:
Same request to use datasource as https://github.com/terraform-google-modules/terraform-docs-samples/pull/157#discussion_r967291181